src/test/resources/test.properties

```diff
@@ -11,6 +11,7 @@ pmd.ruleSets=rulesets/java/android.xml,rulesets/java/basic.xml,rulesets/java/bra
 spotbugs.enabled=true
 spotbugs.includeFilter=
 spotbugs.excludeFilter=
+spotbugs.plugins.location=SpotBugsPlugins
```

Comment:
What is this name? A directory?

Reply:
this is relative path, I've get rid of that in configuration.